src/ape_cache/_cli.py

```diff
@@ -0,0 +1,82 @@
+import click
+import pandas as pd
+
+from ape import networks
+from ape.cli import NetworkBoundCommand, network_option
+from ape.logging import logger
+from ape.utils import ManagerAccessMixin
+
+
+def get_engine():
+    return ManagerAccessMixin.query_manager.engines["cache"]
+
+
+@click.group(short_help="Query from caching database")
+def cli():
+    """
+    Manage query caching database.
+    """
+
+
+@cli.command(short_help="Initialize a new cache database")
+@network_option()
```

Comment:
```suggestion
@network_option(required=True)
```

Reply:
thinking about whether this should actually be an "argument" of just ecosystem and network name, instead of leveraging network_option (since provider is not needed to connect)